tests/all_tests.py

```diff
@@ -95,7 +95,7 @@ def test(suppress_unit_tests, suppress_javascript_unit_tests, unittest_glob):
             command = '%s %s -v' % (python_name, unittest_name)
             stdout, stderr, return_value = console(command)
             print(stderr.decode("utf-8"))
-            if not unittest_good_pattern.search(stderr):
```

Comment:
should the `print` above this line be removed?

Reply:
The print is used to show the output of the unit tests, so it is needed.